src/grid.js

```diff
@@ -13,6 +13,7 @@ import clip from 'liang-barsky';
 const createGrid = (
   { width, height, orderer },
   {
+    cellSize = null,
```

Comment:
As I suggested in my last review: let's replace `itemSize` with `cellSize` instead of handling both. What do we need the `itemSize` in the grid for if we have the `cellSize`? The grid itself shouldn't care about the `itemSize`.

If I am wrong, which can be the case, please point out why the grid needs `itemSize`.